ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -383,6 +395,9 @@ void getAndVerify(BlockingChannel channel, int blobsCount) throws Exception {
           BlobProperties propertyOutput = MessageFormatRecord.deserializeBlobProperties(resp.getInputStream());
           Assert.assertEquals(propertyOutput.getBlobSize(), properties.get(i).getBlobSize());
```

Comment:
Unrelated to the patch, but i think the expected and actual args are flipped in these asserts